src/modals/StartGame.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+	import Dialog from './Dialog.svelte';
+	import { onMount } from 'svelte';
+	let showModal = true;
+	let url = '';
+
+	onMount(() => (url = window.location.href));
```

Comment:
Read up on the page store in sveltekit

Reply:
Read. Will update when doing another fix in different PR.

Reply:
Read. Will update when doing another fix in different PR.